spec/support/tom_select_helper.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module TomSelectHelper
+  def select_tom_select(value, from:)
+    container = find(:id, from)
+
+    within(container) do
+      find('.ts-control').send_keys(value)
```

Comment:
I'm wondering if this is necessary. Doesn't capybara automatically scroll and find the desired element without the need for searching?

If needed, I wonder does the simpler `fill_in '.ts-control' with: value`  work?

Reply:
It does currently, but once we move to the latest version of tom select using `fill_in '.ts-control' with: value` will break.

Reply:
So, I guess we leave this one as is?

PS: I also could not make it work when using `fill_in`:

```
     Failure/Error: fill_in '.ts-control', with: value
     
     Capybara::ElementNotFound:
       Unable to find field ".ts-control" that is not disabled within #<Capybara::Node::Element tag="div" path="//HTML[1]/BODY[1]/DIV[1]/DIV[3]/DIV[1]/DIV[1]/DIV[1]/FORM[1]/DIV[1]/FIELDSET[1]/DIV[1]/DIV[1]/DIV[1]">
```